browser/ui/brave_browser.cc

```diff
@@ -58,20 +61,25 @@ bool BraveBrowser::ShouldUseBraveWebViewRoundedCorners(Browser* browser) {
 }
 
 BraveBrowser::BraveBrowser(const CreateParams& params) : Browser(params) {
+  if (sidebar::CanUseSidebar(this)) {
+    // Below call order is important.
```

Comment:
this seems like a design problem, we should open a follow-up ticket to address this